### PR TITLE
Finish coding the hero animation

### PR DESCRIPTION
**Pull request by cdedreuille**

### CODE REVIEW

File: apps/frontpage/components/home/hero/index.tsx [hunk 1]
```diff
@@ -55,15 +55,32 @@ const Star = ({ x = 0, y = 0, w = 14, delay = 0 }) => {
...
+  useEffect(() => {
+    setSlideInterval();
+    return () => {
+      if (intervalId.current) {
+        clearInterval(intervalId.current);
+      }
+    };
   }, []);
 
+  const handleSlideChange = (newSlide: number) => {
```
Comment (kylegach) [inline apps/frontpage/components/home/hero/index.tsx:79]: I expected it to stop automatically switching between the four tabs once I clicked on one. Wdyt?

File: apps/frontpage/components/home/hero/manager/index.tsx [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+import { PanelControls } from './panel-controls';
+import { AnimatePresence } from 'framer-motion';
+import { Doc } from './doc';
+import { PanelInteractions } from './panel-interactions';
+import { Canvas } from './canvas';
+import { PanelVisualTesting } from './panel-visual-testing';
 
 export const Manager: FC<{ slide: number }> = ({ slide }) => {
   return (
     <div
```
Comment (kylegach) [inline apps/frontpage/components/home/hero/manager/index.tsx:13]: (Not sure where this feedback belongs.)

I find myself missing the animated cursor. It's pretty hard to parse the difference between each feature, as it is.

The simulated interactions for each feature in the current version also did a lot to illustrate them. E.g. navigating between different stories, changing control values. And while it's not in the current one, I think clicking on a couple different interactions steps would be similarly helpful.

File: apps/frontpage/components/home/hero/manager/panel-visual-testing.tsx [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
 
-export const PanelVisualTesting = () => {
+export const PanelVisualTesting = ({ slide }: { slide: number }) => {
   return (
     <motion.div
       initial={{ y: 40, opacity: 0 }}
       animate={{ y: 0, opacity: 1 }}
       exit={{ y: 40, opacity: 0 }}
-      className="absolute bottom-0 left-0 w-full h-[40%] lg:bottom-auto lg:left-auto lg:top-0 lg:right-0 lg:h-full lg:w-[320px] border-t border-t-[#D9E0E6] lg:border-t-0 lg:border-l lg:border-l-[#D9E0E6]"
+      className="absolute bottom-0 left-0 w-full h-[50%] sm:h-[40%] lg:bottom-auto lg:left-auto lg:top-0 lg:right-0 lg:h-full lg:w-[400px] border-t border-t-[#D9E0E6] lg:border-t-0 lg:border-l lg:border-l-[#D9E0E6] flex flex-col"
```
[the remark this answers is not in the document]
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/manager/panel-visual-testing.tsx:19]: This is to show the different possibilities and position of the panel. Also this looks better to show different things moving rather than just at the bottom as you might not see it at first.

File: apps/frontpage/components/home/hero/index.tsx (hunk 1 above)
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/index.tsx:79]: To stop automatically? If you click it reset the timer and will start again after 3 seconds. I don't think we should make it stop forever.

Comment (cdedreuille) [reply apps/frontpage/components/home/hero/index.tsx:79]: To stop automatically? If you click it reset the timer and will start again after 3 seconds. I don't think we should make it stop forever.

File: apps/frontpage/components/home/hero/manager/index.tsx (hunk 2 above)
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/manager/index.tsx:13]: I think we should not confuse 2 things:
- A guided tour of the product
- A presentation of the tool in general

With this update I'm going after the second one so people can understand through the tab and the visual the main features. You don't need to go in details but just understand if this looks "cool" enough or with the right keywords to learn more about it.

The guided tour should be done perhaps in a video as you need people to stick to it. The version we have today is way too slow, incomplete and with no way to come back to a specific point.

File: apps/frontpage/components/home/hero/manager/panel-visual-testing.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/manager/panel-visual-testing.tsx:19]: @kylegach I'm going to resolve this one as the viewport panel looks better on the right side compare to the bottom. I'm working on improving the storytelling though. This might help.

File: apps/frontpage/components/home/hero/index.tsx (hunk 1 above)
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/index.tsx:79]: After some thoughts and discussion you were right. It will look better if we stopped the animation if the user clicks on it. I'll change that.

File: apps/frontpage/components/home/hero/manager/index.tsx (hunk 2 above)
Comment (cdedreuille) [reply apps/frontpage/components/home/hero/manager/index.tsx:13]: We discussed this one on Slack and in our design Jam and you're right, we could introduce some animations on each slide to make it more interactive. This will require more work but it is worth it. I'll work on it.
